api/models/userDao.js

```diff
@@ -0,0 +1,31 @@
+const dataSource = require('./data-source');
+
+const selectUserByEmail = async (email) => {
```

Comment:
의미상 다음과 같이 함수 이름을 변경하는 것이 좋을 것 같습니다.

```suggestion
const getUserByEmail = async (email) => {
```